designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ScenarioActionRepositoryAuditLogDecorator.scala

```diff
@@ -0,0 +1,177 @@
+package pl.touk.nussknacker.ui.process.repository
+
+import db.util.DBIOActionInstances._
+import pl.touk.nussknacker.engine.api.Comment
+import pl.touk.nussknacker.engine.api.deployment.ProcessActionState.ProcessActionState
+import pl.touk.nussknacker.engine.api.deployment._
+import pl.touk.nussknacker.engine.api.process.{ProcessId, ProcessName, ProcessingType, VersionId}
+import pl.touk.nussknacker.ui.process.ScenarioActivityAuditLog
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+
+import java.time.Instant
+import scala.concurrent.ExecutionContext
+
+class ScenarioActionRepositoryAuditLogDecorator(underlying: ScenarioActionRepository)(
```

Comment:
wouldn't it be better to pass the logger instance instead of using ScenarioActivityAuditLog object directly?

Reply:
Does it have any benefits? Maybe I'm missing something, but right now everything related to actual logging is encapsulated in single object, with single logger instance.

Reply:
ScenarioActivityAuditLog looks like sth which is specialized to log using slf4j facade. It could have interface and the current implementation could be one of the several potential ones. But ok ... maybe there is not too much sense doing it now)